charts/kube-prometheus-stack/values.yaml

```diff
@@ -3683,16 +3683,15 @@ prometheus:
     #   matchLabels:
     #     prometheus: somelabel
 
-    ## If true, a nil or {} value for prometheus.prometheusSpec.scrapeConfigSelector will cause the
-    ## prometheus resource to be created with selectors based on values in the helm deployment,
-    ## which will also match the scrapeConfigs created
-    ##
-    scrapeConfigSelectorNilUsesHelmValues: true
-
     ## scrapeConfigs to be selected for target discovery.
-    ## If {}, select all scrapeConfigs
+    ## If matchLabels.release: "{{ $.Release.Name }}" the prometheus resource will be created
```

Comment:
@jkroepke did you have a look at this? Is it correct this way?

Reply:
It looks correct to me for the Selectors. Keep in mind that the namespaceSelector behave a bit differently.

For If null case. also mention that the entire resource is ignored.